### ISSUE TITLE
Strange behavior in the Interval Selection Example

### ISSUE THREAD

**Issue by paduel** [altair-viz/altair#1921]
Selection in the [Interval Selection Example](https://altair-viz.github.io/gallery/interval_selection.html) does not seem to work properly. However, I remember looking at that example a while back and responding correctly. 

I ran the code on my machine and it has the same apparent error. Is it something due to the last update?

**Comment by paduel** [comment]
I got the expected result using this variation of the code.

```import altair as alt
from vega_datasets import data

source = data.sp500.url

brush = alt.selection(type='interval', encodings=['x'])

base = alt.Chart(source).mark_area().encode(
    x='date:T',
    y='price:Q'
).properties(
    width=600,
    height=200
)

upper = base.transform_filter(brush)

lower = base.properties(
    height=60
).add_selection(brush)

upper & lower
```

**Comment by jakevdp** [comment]
This was fixed in #1907

The fix will make it to the website at the next release.
